sys-libs/libcpuid/libcpuid-0.4.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit autotools-utils
+
+DESCRIPTION='A small C library for x86 CPU detection and feature extraction'
+HOMEPAGE="http://${PN}.sourceforge.net"
```

Comment:
Please plaintext it so it's easier to copypaste. Also check if `https://` is available.

Reply:
I am sorry, but `https://` does not seem to be available.